src/Nethermind/Nethermind.DataMarketplace.Consumers.Infrastructure/Rpc/Models/GasPriceDetailsForRpc.cs

```diff
@@ -0,0 +1,37 @@
+//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using Nethermind.DataMarketplace.Core.Services.Models;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Nethermind.DataMarketplace.Consumers.Infrastructure.Rpc.Models
+{
+    public class GasPriceDetailsForRpc
+    {
+        public UInt256 Price { get; set; }
+        public double WaitTime { get; set; }
```

Comment:
why double?

Reply:
Could be decimal, but it shouldn't matter that much (Gas Station API returns wait time in minutes with a single decimal place).